packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -161,18 +161,8 @@ export function getCommonConfig(wco: WebpackConfigOptions) {
 
   let buildOptimizerUseRule;
   if (buildOptions.buildOptimizer) {
-    // Set the cache directory to the Build Optimizer dir, so that package updates will delete it.
-    const buildOptimizerDir = g['_DevKitIsLocal']
-      ? nodeModules
-      : path.dirname(require.resolve('@angular-devkit/build-optimizer', { paths: [projectRoot] }));
-    const cacheDirectory = path.resolve(buildOptimizerDir, './.cache/');
-
     buildOptimizerUseRule = {
       use: [
-        {
-          loader: 'cache-loader',
```

Comment:
Can you also remove `cache-loader` from the dependencies, and comment above?

Reply:
Also done.